shared/common-adapters/plain-input.desktop.js

```diff
@@ -73,6 +75,13 @@ class PlainInput extends React.PureComponent<InternalProps> {
   }
 
   transformText = (fn: TextInfo => TextInfo, reflectChange?: boolean) => {
+    const controlled = !!this.props.value
```

Comment:
shouldn't you check if it's null or undefined instead? since empty string can be a valid value.

Reply:
yup, thanks!